forward_script/src/QueryBuilder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace Wikimedia\ForwardScript;
+
+class QueryBuilder {
+
+	public function getQuery( $info, $pageName, $id = '', $coordinates = [] ) {
+		$category = preg_replace( '/^(:?Category|Kategorie):/', '', $info->category );
+		$query = array_merge( array_filter( $coordinates ), [ 'categories' => $category ] );
+		if ( $id && empty( $info->duplicate_ids ) && empty( $info->id_not_found ) ) {
```

Comment:
If I am getting the whole process (running Python script, converting the result from JSON etc) right this `duplicate_ids` fields is always set to the boolean value. Then the relevant part of the above condition might also written as `... && !$info->duplicate_ids ...`.
The `empty` does consider false to be an empty value but why not be specific if possible?